source/connect/connection-targets.txt

```diff
@@ -0,0 +1,107 @@
+.. _c-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and a ``mongoc_client_t`` structure
```

Comment:
Q: Is "structure" the right term here?

Reply:
I believe C does not use objects, and instead uses "structures", or "structs". In Michael's review of my last C Driver PR, he mentioned to use "structure" when referring to vars of type `mongoc_client_t`, but I can also ask my tech reviewer to confirm!